vsphere/tests/test_unit.py

```diff
@@ -3016,8 +3019,10 @@ def test_property_metrics_metric_filters(
     aggregator.assert_metric('datadog.vsphere.refresh_infrastructure_cache.time')
     aggregator.assert_metric('datadog.vsphere.refresh_metrics_metadata_cache.time')
     aggregator.assert_metric('datadog.vsphere.query_metrics.time')
+    aggregator.assert_metric('vsphere.datacenter.count')
```

Comment:
Is this test about datacenter?

Reply:
No,, but we want to assert that all metrics are covered by assertions so we needed to add a line about datacenters: https://github.com/DataDog/integrations-core/pull/17446/files#diff-a66e7a8e26e8d7cf8f587cf002a7fed3eb836c3ec51680baf361e5a5a1e5457bR3026